automation/capitanodoc/capitanodoc.ts

```diff
@@ -0,0 +1,152 @@
+/**
```

Comment:
The `capitanodoc.ts` file already existed but was previously located at the root of the CLI's repo. I thought it made more sense to move it under `automation/capitanodoc/` (a folder that also already existed).
(Maybe it was meant to be a Capitano convention to have `capitanodoc.ts` at the root? But it was a half code / half markdown content file, strange to be at the root.)